From 9defbc269e8402b048dd344ece6af5c3694d693d Mon Sep 17 00:00:00 2001
From: Daniel Mueller <deso@posteo.net>
Date: Tue, 16 Feb 2021 08:36:57 -0800
Subject: [PATCH] Stabilize new resolver

With this change we mark the new resolver as stable, allowing its use on
the stable channel. We require this new version because the old one
does not necessarily determine feature sets correctly for dependencies
that are both unconditional and conditional, but with differing
features. Also see https://github.com/rust-lang/cargo/issues/9176.
---
 src/cargo/core/features.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/cargo/core/features.rs b/src/cargo/core/features.rs
index 487abc..caa8d4 100644
--- a/src/tools/cargo/src/cargo/core/features.rs
+++ b/src/tools/cargo/src/cargo/core/features.rs
@@ -210,7 +210,7 @@ features! {
         [unstable] named_profiles: bool,
 
         // Opt-in new-resolver behavior.
-        [unstable] resolver: bool,
+        [stable] resolver: bool,
 
         // Allow to specify whether binaries should be stripped.
         [unstable] strip: bool,
-- 
2.26.2

